packages/mobile/src/screens/DeleteChannel/DeleteChannel.screen.tsx

```diff
@@ -10,7 +10,7 @@ import { navigationSelectors } from '../../store/navigation/navigation.selectors
 export const DeleteChannelScreen: FC<DeleteChannelScreenProps> = ({ route }) => {
   const dispatch = useDispatch()
 
-  const { channel } = route.params
+  const { channel: channelName } = route.params
```

Comment:
How does it work? It looks like 'channelName' was supposed to be a type definition

Reply:
This is destructuring and renaming property
Example: https://flaviocopes.com/how-to-rename-object-destructuring/